R/model_parameters.psych.R

```diff
@@ -140,16 +149,43 @@ model_parameters.fa <- model_parameters.principal
 
 #' @rdname model_parameters.principal
 #' @export
-model_parameters.omega <- function(model, sort = FALSE, threshold = NULL, labels = NULL, ...) {
-
-  # TODO: TO BE SERVERLY IMPROVED
-  # https://github.com/cran/psych/blob/master/R/print.psych.omega.R
-
-  # Get loadings
-  loadings <- as.data.frame(unclass(model$schmid$sl))
-
-  # Format
-  loadings <- cbind(data.frame(Variable = row.names(loadings)), loadings)
-  row.names(loadings) <- NULL
-  loadings
+model_parameters.omega <- function(model, ...) {
+  
+  # Table of omega coefficients
+  table_om <- model$omega.group
+  colnames(table_om) <- c("Omega_Total", "Omega_Hierarchical", "Omega_Group")
+  table_om$Composite <- row.names(table_om)
+  row.names(table_om) <- NULL
+  table_om <- table_om[c("Composite", names(table_om)[names(table_om) != "Composite"])]
+  
+  # Get summary: Table of Variance
+  table_var <- as.data.frame(unclass(model$omega.group))
+  table_var$Composite <- rownames(model$omega.group)
+  table_var$Total <- table_var$total*100
```

Comment:
Please use whitespaces around `*` etc.: `table_var$total*100` -> `table_var$total * 100`

Reply:
@zen-juen you can run `styler::style_file("filepath")` to automatically nicely format a given file

Reply:
I don't like these tools, as they do too much in my opinion. Would be sufficient to activate style diagnostic (https://support.rstudio.com/hc/en-us/articles/205753617-Code-Diagnostics) and then just hit Ctrl+Shift+A on selected lines. :-)

Reply:
I think they are ways of parametrizing styler, but yeah I can understand (although as for now I've been fairly satisfied with styler's defaults ☺️)